ChangeLog.md

```diff
@@ -125,6 +125,8 @@
 
   * Change size limit for form data to account for increase from encoding (Nathan Bowman).
 
+  * Change styling for invalid input and `pl-string-input` for increased clarity. (Nicolas Nytko)
```

Comment:
Probably should add a few more entries as this is pretty large... e.g. 

- `allow-fractions` in `pl-number-input` !?!? 
- `missing-input` when question is changed by course staff...

Reply:
Changelog has been deprecated according to @mwest1066 in favour of git history